src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -68,7 +68,7 @@ const ExplorationViewPage = ({
         <HeatmapPlot experimentId={experimentId} width={width} height={height} />
       ),
     },
-    Tools: {
+    Genes: {
```

Comment:
These changes are done to replace `Tools` with `Genes` in the data exploration display.